usaspending_api/common/threaded_data_loader.py

```diff
@@ -34,10 +36,13 @@ class ThreadedDataLoader():
     #   pre_row_function - Like post_row_function, but before the model class is updated
     #   post_process_function - A function to call when all rows have been processed, uses the same
     #                           function parameters as post_row_function
-    def __init__(self, model_class, threads=10, field_map={}, value_map={}, collision_field=None, collision_behavior='update', pre_row_function=None, post_row_function=None, post_process_function=None):
+    def __init__(self, model_class, processes=-1, field_map={}, value_map={}, collision_field=None, collision_behavior='update', pre_row_function=None, post_row_function=None, post_process_function=None):
```

Comment:
Re using `{}` and `[]` in the constructors -- that's why it hasn't bitten yet, but it's something to keep in mind for the future. It's easy to imagine doing something seemingly innocuous like

```python
# file1.py
loader = ThreadedDataLoader(some_model_class)
loader.field_map['another_field'] = "some value"
do_something_with(loader)

# file2.py
loader = ThreadedDataLoader(some_other_model)
assert loader.field_map['another_field'] == "some value"  # wait what?
```